templates/web/base/reports/_other_districts-list.html

```diff
@@ -13,7 +13,8 @@ <h2>[% parish_text %]</h2>
         [% IF child.type == 'CPC' %]
         <li class="item-list__item item-list--wards__item">
           <label class="[% show_parish_shown_class %]">
-            <input type="checkbox" name="ward" value="[% child.name | html %]">
+            [% SET value_type = child.name _ "?type=" _ child.type %]
+            <input type="checkbox" name="ward" value="[% value_type | html %]">
```

Comment:
Hmm, why does this base file exist at all, if it's only for Oxfordshire? The only differences appear to be bug fixes in the Oxfordshire version - can the base file be removed entirely?